synesdeija/festivalsWiki
# Bump mongo deps to latest

# Purpose

 



This PR bumps MongoDB, mongoose, mongosh, and connect-mongo to the latest versions and resolves breaking changes from that activity.

The dependabot bump for connect-mongo in PR #14 introduced breaking changes to the application that needed to be resolved. In sum, connect-mongo v4+ deprecated the `mongooseConnection` value. 

To fix the app, I followed [connect-mongo's V4 migration guide](https://github.com/jdesboeufs/connect-mongo/blob/HEAD/MIGRATION_V4.md) as well as [this stack overflow answer.](https://stackoverflow.com/a/73210655)

# Testing

 


- Check out this branch `git fetch && git checkout AOL/bump-mongo`
- Install updated packages `npm install`
- Start the server: `npm start`
- Try to complete some database-oriented task (such as logging in) 

# Documentation updates

None 🙂
